{{cookiecutter.github_repository}}/provisioner/roles/nginx/defaults/main.yml

```diff
@@ -5,7 +5,7 @@ nginx_worker_connections: 1024
 nginx_client_max_body_size: '10M'
 
 {% raw %}
-nginx_conf_file_name: '{{ project_name }}-{{ deploy_environment }}'
+nginx_conf_file_name: '{{ project_namespace }}'
```

Comment:
How would we differentiate `dev` and `qa` configs when they are on the same server?

Reply:
Actually, nevermind. I got it :)